EIPS/eip-747.md

```diff
@@ -64,51 +52,44 @@ If the asset is already listed in the user's wallet, the wallet SHOULD re-fetch
 - Downloading the image failed to load
 - The wallet blocked the asset, either due to an allowlist or denylist
 
-## Rationale
+### `ERC1046` type
 
-Displaying a user's assets is a basic feature that every modern dapp user expects. However, keeping this list, and polling for it from the network can be costly, especially on bandwidth constrained devices.
+The format of the options field is:
 
-Most wallets today either manage their own assets list, which they store client side, or they query a centralized API for balances, which reduces decentralization, letting that API's owner easily correlate account holders with their IP addresses.
+```typescript
+interface ERC1046WatchAssetOptions {
+{
+    address: string; // The hexadecimal Ethereum address of the token contract
+    chainId?: number; // The chain ID of the asset. If empty, defaults to the current chain ID.
+  };
+}
+```
 
-Maintaining one of these assets lists becomes a political act, and maintainers can be subject to regular harassment and pressure to list otherwise unknown assets.
+`address` is required, and the other fields are optional. `address` MUST be the hexadecimal Ethereum address of the token contract. `chainId` MUST be the chain ID to which the asset belongs.
```

Comment:
Given that we are including `chainId`, it feels like this EIP is intended to work on chains other than Ethereum mainnet+testnets, so recommend removing reference to `Ethereum address`.
```suggestion
`address` is required, and the other fields are optional. `address` MUST be the hexadecimal address of the token contract. `chainId` MUST be the chain ID to which the asset belongs.
```

Reply:
I agree with this change.